tests/test_fetcher_ng.py

```diff
@@ -94,7 +94,7 @@ def test_fetch_in_chunks(self) -> None:
     # Incorrect URL parsing
     def test_url_parsing(self) -> None:
         with self.assertRaises(exceptions.DownloadError):
-            self.fetcher.fetch("missing-scheme-and-hostname-in-url")
+            self.fetcher.fetch("http://invalid/")
```

Comment:
this changes the underlying error (`No address associated with hostname` vs `Name or service not known`) but that seems fine.